### ISSUE TITLE
Black Text Style Tweak

### ISSUE THREAD

**Issue by NotAFile** [koreader/koreader#6018]
* KOReader version: 2020.03.2
* Device: Kobo Aura ONE

# Issue

I've created the following style tweak, because I've encountered epubs with unhelpful font colors:

```css
body p { color: black !important; }
```

I'd like to see this added to the built-in style tweaks for the people that don't know CSS. However, I'm unsure what section of the tweaks they'd belong in, or what the general conventions around these kinds of tweaks are in the koreader project.

**Comment by mergen3107** [comment]
In general, to tweak css:
From PC, go to koreader/data folders. There, copy epub.css to a new file, e.g. my_epub.css (to save your changes after updates). Add your tweaks to `p {` . Then go to KOReader on your device, and select this css in Styles

**Comment by NiLuJe** [comment]
@mergen3107: That wasn't really the question ;).

@NotAFile: @Frenzie and/or @poire-z should be able to tell you more about how the Tweaks menu are laid out ;).

**Comment by poire-z** [comment]
We already have one (more radical), at the bottom of `Miscellaneous>`
https://github.com/koreader/koreader/blob/2945a2fb79bfb73670363a1fe4ac7f0a1ee0c2c4/frontend/ui/data/css_tweaks.lua#L793-L805

**Comment by NotAFile** [comment]
@poire-z I can't believe I didn't spot that. That should be good enough. I think anything more specific should really be done on a per-epub level.

**Comment by poire-z** [comment]
(Might be @mergen3107 hasn't yet discovered the wonder of our [Style tweaks](https://github.com/koreader/koreader/wiki/Style-tweaks) :tm: :)

**Comment by mergen3107** [comment]
Oops, sorry for misinterpreting the question :)
@poire-z , I use them mostly on defaults because I use my own [fb2_Kollindle.css.txt](https://github.com/koreader/koreader/files/4424044/fb2_Kollindle.css.txt) file which works great for fb2.zip files (and surprisingly good for epubs too), which I made for fb2->azw3 conversion a long time ago for Kindle. More details on the style [here](http://4pda.ru/forum/index.php?showtopic=942254&st=20#entry83191059), sorry it's in Russian but it also has screenshots.
Basically, I wanted a good indentation, good looking poems, footnote superscripts that don't spoil line spacing, and more compact text layout to fit more text on a page.
The only thing I miss is "chapter vignettes", pieces fo decoration around h1-h3 headings to make them pretty (the screenshots in the above linked post can show them), but I'm not sure if this is an easy implementation.
